src/databricks/labs/ucx/config.py

```diff
@@ -44,6 +46,8 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     # Flag to see if terraform has been used for deploying certain entities
     is_terraform_used: bool = False
 
+    grant_migration_strategy: AclMigrationWhat | None = None
```

Comment:
i don't want to add any unstable enums to WorkspaceConfig, because it'll escalate when we change the enum and break customer configuration loading from previous formats. 

please rewrite the code NOT to use enums in config? you assume that ACLs are `either ... or ...`, but they have to be `... and ...`

![image](https://github.com/databrickslabs/ucx/assets/259697/2231040e-3768-4dfa-b5c5-11e1ca9bd768)